source/backend/routes/users.js

```diff
@@ -86,4 +88,45 @@ router.delete('/:savedRecipeId', async (req, res) => {
   }
 });
 
+/* POST /auth/signup */
+/* Creates new user in database with username, password and email -> unique user/email 
+required */
+router.post('/auth/signup', async (req, res) => {
+  try{
+    const { username } = req.username;
+    const { password } = req.password;
+    const hashedPwd = await bcrypt.hash(password, 10);
```

Comment:
After changing the library to bcrypt-nodejs, you can just do: `bcrypt.hashSync(user.password, null, null);` with salt specified as null for simplicity. No need to await either bc hashSync is not an async function